### PR TITLE
await in AuthorManager.js

### PR DESCRIPTION
**Pull request by webzwo0i**
Should partially fix https://github.com/ether/etherpad-lite/issues/5000

createGroupIfNotExistsFor/createGroup seems to be using the correct code already, so I'm not sure yet what happened

### CODE REVIEW

**Comment by webzwo0i** [comment]
making draft, as it breaks other tests. The new test runs, but other tests that need the database fail now. (maybe the delayed set is still run?)

**Comment by webzwo0i** [comment]
I had code in that checked for the size of db.buffer._cache before the test runs. It showed that there was an globalAuthor entry (probably from one of the last tests?). After the test finishes, I check db.buffer._cache again to see if there are now two values inside (1 from before and 1 from running my test). Unexpectedly, there is only one!

File: src/tests/backend/specs/regression-db.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  });
+
+  after(async function () {
+    // ensure db is closed after all writes have been written
+    await waitForDB();
+    db.set = tmp;
+    if (standalone) {
+      await db.shutdown();
+    }
+  });
```
Comment (rhansen) [inline src/tests/backend/specs/regression-db.js:55]: None of this is necessary if you do `await common.init();`.

Comment (webzwo0i) [reply src/tests/backend/specs/regression-db.js]: I do use `common.init()` now and removed everything except `db.set = tmp` to undo the delayed set. Thanks for pointing that out!

**Comment by rhansen** [comment]
I pushed a couple of fixup commits; please take a look. If they look OK to you I will squash them into their respective commits and merge.

**Comment by webzwo0i** [comment]
looking great!

**Pull request merged**